src/Uno.Templates/Uno.Templates.csproj

```diff
@@ -47,6 +47,15 @@
 
 		<RemoveDir Directories="$(TemplateContentFolder)" />
 
+		<!-- Ensure we keep the .gitignore up to date with the latest changes to the official VisualStudio.gitignore -->
+		<DownloadFile
```

Comment:
Can we move this to a different PR  - this is something that can go forward regardless

Reply:
now that we're moving this to ready for review this should be fine to leave here